### PR TITLE
(Publish on 09/25/2023) Android agent v7.1.0 release notes

### PR DESCRIPTION
**Pull request by ywang-nr**
Ticket: https://issues.newrelic.com/browse/NR-160141
* What problems does this PR solve? - Android agent release notes

### CODE REVIEW

**Comment by ywang-nr** [comment]
@ndesai-newrelic for hybrid review

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-710.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+subject: Android agent
+releaseDate: '2023-09-21'
+version: 7.1.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.1.0.zip'
+---
+
+<Callout variant="important">
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-710.mdx:8]: We can remove this now that 7.0.0 has shipped

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-710.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  Android agent 7.x is increasing the minimum Gradle version to 7.1. Customers will be required to upgrade to Android Gradle Plugin (AGP) 7.0.0 and higher.
+</Callout>
+
+## New
+
+* Adds support for DexGuard 9+ with Gradle 8/AGP 8
+* Adds unity support with custom exceptions
+* Adds map parameters for both custom and auto HTTP instrumentation to ensure data accuracy
+* Decouple map tagging with uploads in plugin to improve the build pipeline by making the tagged map a cachable build artifact
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-710.mdx:21]: That is very good!

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-710.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+
+## Fixed
+
+* Fix NoClassDefFoundError when using agent v7.0.0 due to SHA signed libraries are not instrumented
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-710.mdx:26]: How about, "
* Fix NoClassDefFoundError when using agent v7.0.0 due to uninstrumented signed libraries"?

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-710.mdx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+* Fix an issue that excludeVariantInstrumention is not taking camal case variants
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-710.mdx:27]: "... not accepting camel case variant names"

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-710.mdx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+* Fix an issue with obfuscated attribute due to field name change
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-710.mdx:28]: This is entirely internal and can be removed.

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]
Awesome.

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**